diff -u -r -x .DS_Store upstream/pkgs/duoauthproxy/duoauthproxy/lib/util.py patched/pkgs/duoauthproxy/duoauthproxy/lib/util.py
--- upstream/pkgs/duoauthproxy/duoauthproxy/lib/util.py	2020-11-19 19:18:28.000000000 -0500
+++ patched/pkgs/duoauthproxy/duoauthproxy/lib/util.py	2020-12-11 21:10:36.000000000 -0500
@@ -195,7 +195,7 @@
         # This local file is <AUTHPROXY_INSTALL_DIR>/usr/local/lib/python2.7/site-packages/duoauthproxy/lib/util.py.
         # Find <AUTHPROXY_INSTALL_DIR>.
         _home_dir = os.path.abspath(
-            os.path.join(os.path.dirname(__file__), *[".."] * 7)
+            os.path.join(os.path.dirname(__file__), *[".."] * 5)
         )
 
 
diff -u -r -x .DS_Store upstream/pkgs/duoauthproxy/scripts/authproxy_update_sso_enrollment_code patched/pkgs/duoauthproxy/scripts/authproxy_update_sso_enrollment_code
--- upstream/pkgs/duoauthproxy/scripts/authproxy_update_sso_enrollment_code	2020-11-19 19:18:28.000000000 -0500
+++ patched/pkgs/duoauthproxy/scripts/authproxy_update_sso_enrollment_code	2020-12-11 21:12:55.000000000 -0500
@@ -10,10 +10,7 @@
 from duoauthproxy.lib import const, log, util
 from duoauthproxy.lib.secret_storage import get_storage_filename, store_secret
 
-# --- install script will customize this ---
-USER_DEFAULT = None
-# ------------------------------------------
-
+USER_DEFAULT = 'root'
 
 def set_linux_permissions():
     import pwd
diff -u -r -x .DS_Store upstream/pkgs/duoauthproxy/scripts/authproxyctl patched/pkgs/duoauthproxy/scripts/authproxyctl
--- upstream/pkgs/duoauthproxy/scripts/authproxyctl	2020-11-19 19:18:28.000000000 -0500
+++ patched/pkgs/duoauthproxy/scripts/authproxyctl	2020-12-11 21:13:57.000000000 -0500
@@ -89,9 +89,7 @@
 
 class LinuxAuthproxyController(AuthproxyController):
 
-    # --- install script will customize this ---
-    USER_DEFAULT = None
-    # ------------------------------------------
+    USER_DEFAULT = 'root'
 
     # linux standard status codes for 'status' initscript command
     STATUS_RUNNING = 0
@@ -107,7 +105,7 @@
             python_root = os.path.abspath(root_relative)
 
         if duo_root is None:
-            root_relative = os.path.join(os.path.dirname(script_abspath), *[".."] * 3)
+            root_relative = os.path.join(os.path.dirname(script_abspath), "..")
             duo_root = os.path.abspath(root_relative)
 
         super(LinuxAuthproxyController, self).__init__(duo_root)
diff -u -r -x .DS_Store upstream/pkgs/duoauthproxy/scripts/duoauthproxy.tap patched/pkgs/duoauthproxy/scripts/duoauthproxy.tap
--- upstream/pkgs/duoauthproxy/scripts/duoauthproxy.tap	2020-11-19 19:18:28.000000000 -0500
+++ patched/pkgs/duoauthproxy/scripts/duoauthproxy.tap	2020-12-11 21:14:23.000000000 -0500
@@ -1,8 +1,6 @@
 from duoauthproxy import proxy
 
-# --- install script will customize this ---
-USER_DEFAULT = None
-GROUP_DEFAULT = None
-# ------------------------------------------
+USER_DEFAULT = 'root'
+GROUP_DEFAULT = 'root'
 
 application = proxy.create_application(twistd_user=USER_DEFAULT, log_group=GROUP_DEFAULT)
diff -u -r -x .DS_Store upstream/pkgs/duoauthproxy/setup.py patched/pkgs/duoauthproxy/setup.py
--- upstream/pkgs/duoauthproxy/setup.py	2020-11-19 19:18:28.000000000 -0500
+++ patched/pkgs/duoauthproxy/setup.py	2020-12-11 21:15:20.000000000 -0500
@@ -153,11 +153,11 @@
         scripts=[
             "scripts/authproxy",
             "scripts/authproxyctl",
-            "scripts/install",
             "scripts/authproxy_connectivity_tool",
             "scripts/authproxy_primary_only",
             "scripts/authproxy_support",
             "scripts/authproxy_update_sso_enrollment_code",
+            "scripts/duoauthproxy.tap",
         ],
         **common_metadata
     )
